src/games/controller.ts

```diff
@@ -0,0 +1,88 @@
+import {
+  JsonController, Authorized, CurrentUser, Post, Param, BadRequestError, HttpCode, NotFoundError, ForbiddenError, Get,
+  Body, Patch
+} from 'routing-controllers'
+import User from '../users/entity'
+import { Game, Player, Board } from './entities'
+import {IsBoard} from './gameLogic'
+import { Validate } from 'class-validator'
+import {io} from '../index'
+
+class GameUpdate {
+
+  @Validate(IsBoard, {
+    message: 'Not a valid board'
```

Comment:
When will we need this? Probably at some point but just want to leave a comment that I don't understand this part (yet)

Reply:
This will help to check that a given board is valid before updating the data